packages/interactor/src/specification.ts

```diff
@@ -13,23 +13,37 @@ export type FilterObject<T, E extends Element> = {
   default?: T;
 }
 
+export type ChildObject<T extends Array<any>, E extends Element, F extends Filters<E>, A extends Actions<E>, C extends Children> = {
+  find: (interactor: Interactor<Element, {}, {}, {}>, ...args: T) => InteractorInstance<E, F, A, C>
+}
```

Comment:
This has to be an object so we can disambiguate it from the interactor constructor, which is a function.